QuangF.NV/lession_1/lib/constant.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+const persimmon = Color(0xFFFF6D46);
+const radicalRed = Color(0xFFFC2D64);
+const double heightButton = 66;
+const double radiusButton = 30;
+const double bodyFontSize = 30;
+const double smallFontSize = 15;
+const double smallFontLetterSpacing = 2;
+
+class SignInScreenText {
+  static const signIn = 'SIGN IN';
```

Comment:
e để chung vào 1 class StringRes là được, như này cũng clear, mà nếu nhiều màn hình  thì cũng kha khá class
```suggestion
/// For sign in screen
  static const signIn = 'SIGN IN';
```